WalletWasabi.Fluent/ViewModels/Navigation/TargettedNavigationStack.cs

```diff
@@ -55,6 +55,11 @@ protected override void OnPopped(RoutableViewModel page)
 	{
 		base.OnNavigated(oldPage, oldInStack, newPage, newInStack);
 
+		if (oldPage is { } && oldPage != newPage)
+		{
+			oldPage.IsActive = false;
+		}
+
```

Comment:
This is not related anymore to this PR, but in general, it was a bug. I can remove and add it in a separate PR if necessary.